packages/pigeon/run_tests.sh

```diff
@@ -199,6 +199,8 @@ test_command_line() {
     | grep "public class Message">/dev/null
   # Test dartOut in ConfigurePigeon overrides output.
   $run_pigeon --input pigeons/configure_pigeon_dart_out.dart 1>/dev/null
+  # Make sure AST generation exits correctly.
```

Comment:
Not crashing is useful, but there should also be at least one simple test that it outputs the expected data. AFAICT you could make `generateAst` a no-op and all your tests would still pass.

Reply:
Done, added simple test.  I didn't really want to test it too heavily since I wanted the flexibility to change the format and its output is inconsequential in that it is just for debugging and explicitly not for production.